google_maps/google_maps.py

```diff
@@ -41,9 +40,15 @@ class GoogleMapsXBlock(XBlock):
         scope=Scope.settings
     )
 
-    iframe_height = Integer(
-        display_name=_('Map height'),
-        default=450,
+    place_id = String(
+        display_name=_('Google Maps place id'),
+        default='ChIJCwHM0mF344kRAWE3Rmv-RcY',  # ExtensionEngine, Boston headquarters
+        scope=Scope.settings
+    )
+
+    place_name = String(
+        display_name=_('Search for a place'),
+        default='',
```

Comment:
I would rather not change this because it will affect a few things and I don't really feel like changing everything else just to connect those into one variable.